src/SourceBuild/content/eng/tools/init-build.proj

```diff
@@ -116,6 +117,14 @@
     </Touch>
   </Target>
 
+  <Target Name="BuildSdkArchiveDiff" >
```

Comment:
This target should probably have the same condition as the targets import in build.proj so that the build task only gets built when it is used.